recipes/openlineage-sql/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "openlineage-sql" %}
+{% set version = "0.30.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/openlineage_sql-{{ version }}.tar.gz
+  sha256: df146cb536d6d9a336afe073e73982477ef0dd1220e27975687355034e7d2daa
+
+build:
+  skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
```

Let's see if this fixes the warnings/errors we are getting.

```suggestion
    - {{ compiler('c') }}
    - {{ compiler('rust') }}
```